ChildProject/annotations.py

```diff
@@ -703,7 +709,7 @@ def rename_set(
         self.write()
 
     def merge_annotations(
-        self, left_columns, right_columns, columns, output_set, input
+        self, left_columns, right_columns, columns, output_set, input, skip_existing: bool = False
```

Comment:
so by default it will overwrite, correct?

Reply:
Yes, I think by default it is better than a 'merge' re-generates the converted annotations, this will update the merged set if there were changes to the original sets. So not updating the existing converted files should be a deliberate choice by the user.